### PR TITLE
feat: add temperature command targetting climate entities

### PR DESCRIPTION
**Pull request by mrene**
Adds a command to set the temperature of any climate entities. It uses the temperature units that are configured in home assistant (the api just takes a float)

```shell
❯ hctl temperature heat_pump <TAB>
heat_pump  nest  refrigerator_freezer  refrigerator_fridge
```

```shell
❯ hctl temperature heat_pump 21.5
 SUCCESS  heat_pump temperature set to 21.5
```

### CODE REVIEW

File: cmd/temperature.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	"github.com/xx4h/hctl/pkg"
+	o "github.com/xx4h/hctl/pkg/output"
+	"io"
+)
+
+func newTemperatureCmd(h *pkg.Hctl, out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "temperature",
+		Short:   "Set the temperature of a climate entity",
+		Aliases: []string{"t"},
```
Comment (xx4h) [inline cmd/temperature.go:28]: Alias `t` is aready used by `toggle` command. Maybe `te` and `temp`?

File: pkg/hctltest/testdata/heating_climate_set_temperature_response.json [hunk 2]
```diff
@@ -0,0 +1 @@
+[]
```
Comment (xx4h) [inline pkg/hctltest/testdata/heating_climate_set_temperature_response.json:1]: I know that there's sometimes empty responses from the Home Assistant API. This is also the reason, why `hctl` doesn't really check the response at the moment. Do you really always get an empty response or could you maybe provide an example response for when the API is responding like expected?

File: cmd/temperature.go (hunk 1 above)
Comment (xx4h) [inline cmd/temperature.go:28]:
```suggestion
		Aliases: []string{"te", "temp"},
```

**Review by xx4h — CHANGES_REQUESTED** [review]

**Comment by xx4h** [comment]
Thanks for contributing :heart:

File: pkg/hctltest/testdata/states.json [hunk 3]
```diff
@@ -283,5 +283,35 @@
...
+      "hvac_modes": [
+        "heat",
+        "off"
+      ],
+      "min_temp": 10,
+      "max_temp": 32,
+      "preset_modes": [
+        "none",
+        "eco"
+      ],
```
Comment (xx4h) [inline pkg/hctltest/testdata/states.json:300]:
```suggestion
      "preset_modes": ["none", "eco"],
```

File: pkg/hctltest/testdata/states.json [hunk 4]
```diff
@@ -283,5 +283,35 @@
...
     }
+  },
+  {
+    "entity_id": "climate.heating",
+    "state": "heat",
+    "attributes": {
+      "hvac_modes": [
+        "heat",
+        "off"
+      ],
```
Comment (xx4h) [inline pkg/hctltest/testdata/states.json:294]:
```suggestion
      "hvac_modes": ["heat", "off"],
```

**Review by xx4h — CHANGES_REQUESTED** [review]
Regarding the `JSON_PRETTIER`

File: pkg/hctltest/testdata/heating_climate_set_temperature_response.json (hunk 2 above)
Comment (mrene) [reply pkg/hctltest/testdata/heating_climate_set_temperature_response.json:1]: I'm always getting an empty response when calling `climate.set_temperature` - they don't seem to return the entity's state at all.

Comment (xx4h) [reply pkg/hctltest/testdata/heating_climate_set_temperature_response.json:1]: ok, thanks for checking

File: cmd/temperature.go [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/xx4h/hctl/pkg"
+	o "github.com/xx4h/hctl/pkg/output"
+	"io"
```
Comment (xx4h) [inline cmd/temperature.go:21]:
```suggestion
	"io"

	"github.com/spf13/cobra"

	"github.com/xx4h/hctl/pkg"
	o "github.com/xx4h/hctl/pkg/output"
```

**Review by xx4h — CHANGES_REQUESTED** [review]

**Comment by xx4h** [comment]
Thanks again!

**Pull request merged**